test/integration_tests/long/test_load_executor.py

```diff
@@ -500,6 +501,15 @@ def test_load_pdfs(self):
         self.assertEqual(len(result.columns), 4)
         self.assertEqual(len(result), 26)
 
+    def test_load_query_incorrect_fileFormat(self):
+        try:
+            execute_query_fetch_all(
+                self.evadb,
+                f"""LOAD document '{EvaDB_ROOT_DIR}/data/documents/*.pdf' INTO pdfs;""",
+            )
+        except ExecutorError as e:
```

Comment:
Nits: we can use `with self.assertRaise` to check the error message.
Example: https://github.com/georgia-tech-db/evadb/blob/ea6bfc66f5428bb261f6a6687521a842d66108f9/test/unit_tests/executor/test_create_udf_executor.py#L72

Reply:
Sure, changed it as per the suggestion